kore/app/exec/Main.hs

```diff
@@ -394,6 +419,58 @@ parserInfoModifiers =
                 \in PATTERN_FILE."
     <> header "kore-exec - an interpreter for Kore definitions"
 
+showKoreExecOptions :: KoreExecOptions -> String
+showKoreExecOptions
+    ( KoreExecOptions
+        definitionFileName
+        patternFileName
+        outputFileName
+        mainModuleName
+        smtTimeOut
+        smtPrelude
+        smtSolver
+        breadthLimit
+        depthLimit
+        strategy
+        koreLogOptions
+        koreSearchOptions
+        koreProveOptions
+        koreMergeOptions
+        rtsStatistics
+        _
+    )
+  =
+    unlines
+        [ "KoreExecOptions:"
+        , "definitionFileName = " <> show definitionFileName
+        , "patternFileName = " <> show patternFileName
+        , "outputFileName = " <> show outputFileName
+        , "mainModuleName = " <> show mainModuleName
+        , "smtTimeOut = " <> show smtTimeOut
+        , "smtPrelude = " <> show smtPrelude
+        , "smtSolver = " <> show smtSolver
+        , "breadthLimit = " <> show breadthLimit
+        , "depthLimit = " <> show depthLimit
+        , "strategy = " <> show (fst strategy)
+        , "koreLogOptions = " <> show koreLogOptions
+        , "koreSearchOptions = " <> show koreSearchOptions
+        , "koreProveOptions = " <> show koreProveOptions
+        , "koreMergeOptions = " <> show koreMergeOptions
+        , "rtsStatistics = " <> show rtsStatistics
+        ]
+
+writeOptionsAndKoreFiles :: KoreExecOptions -> IO ()
+writeOptionsAndKoreFiles
+    opts@KoreExecOptions { definitionFileName , patternFileName, outputFileName}
+  = do
+    createDirectoryIfMissing False "./report"
+    writeFile "./report/KoreExecOptions.txt" . showKoreExecOptions $ opts
```

Comment:
This should write a shell script instead, so that after unpacking the bug report archive, we can run `./kore-exec.sh` and it will reproduce exactly what the user did.